namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
     type: integer
     description: >
       Build ID of the CI job.
 
-  - name: build_timestamp
+  - name: timestamp
```

Comment:
this field shows the time for ci job build. While the @timestamp at the top level could show a different time. For example, if the job has the test cases, the @timestamp will be the time for running testcase.